### PR TITLE
Jordan/lighthouse ci

### LINKED ISSUE #3058: add lighthouse metrics to build process
**Is your feature request related to a problem? Please describe.**

- key metrics like performance, SEO, and accessibility seem to be fluctuating. let's add something to help us track this overtime.

**Describe the solution you'd like**

- https://github.com/marketplace/actions/lighthouse-audit

### PR DESCRIPTION
**Pull request by jbibla**
Closes #3058

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+on: [push, pull_request]
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Audit Netlify deploy preview
+      uses: jakejarvis/lighthouse-action@master
+      with:
+        netlify_site: 'develop--lunieio.netlify.com/'
```
Comment (jbibla) [inline .github/workflows/workflow.yml]: @faboweb i can't access netlify - we need a different URL 

https://github.com/marketplace/actions/lighthouse-audit#pull-request-audits-with-netlify-deploy-preview

Comment (faboweb) [reply .github/workflows/workflow.yml]:
```suggestion
        netlify_site: 'lunieio.netlify.com'
```

Comment (faboweb) [reply .github/workflows/workflow.yml]: > i can't access netlify

let's fix that

**Comment by jbibla** [comment]
note: add budget so the CI check is meaningful

File: .github/workflows/workflow.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Lighthouse Audit
+on: [push, pull_request]
```
Comment (faboweb) [inline .github/workflows/workflow.yml]:
```suggestion
on: [pull_request]
```

Comment (faboweb) [reply .github/workflows/workflow.yml]: push doesn't seem to work. does `pull_request` rerun when the branch updates though?

**Comment by jbibla** [comment]
once best practices are in place i don't think we'll actually need this.

**Pull request closed without merging**